enhancements/.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+<!-- 
+	Please fill out the following questions when adding/updating an enhancement in the openshift/enhancements repo
+-->
+### Enhancement Description
+
+<!-- short description of enhancement-->  
+- Brief (1-2 lines) enhancement description:
+
+<!-- link to related discussions about this enhancement, for example a github issue, RFE or public mailing list link if available -->
```

Comment:
This is redundant with the template metadata header. Do you think we need it in both places?

Reply:
I think PRs having a standard format with a brief description is always good practice.

Reply:
Sorry, I chose a bad line for that comment. I meant the discussion links are similar to the tracking link in the metadata.